public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FEC Corgis</title>
+</head>
+
+<body>
+    <div id="root"></div>
+    <div id="root2"></div>
```

Comment:
@manedurphy Are these divs for your primary and secondary module? How are you connecting them to your actual repos modules? I was looking for another reference to root1 and root2 and couldn't find it in the server routes. Did you just rename the getElementById reference to 'root' in your index.jsx?